LICENSE.spdx

```diff
@@ -0,0 +1,6 @@
+SPDXVersion: SPDX-2.1
+DataLicense: CC0-1.0
+PackageName: EPIC
+PackageOriginator: Wouter Deconinck, Sylvester Joosten
+PackageHomePage: https://github.com/eic/epic
+PackageLicenseDeclared: LGPL-3.0-or-later
```

Comment:
Suggested changes for compliance with [the validator](https://tools.spdx.org/app/validate/):
- add `DocumentNamespace` (used a [generated UUID](https://www.uuidgenerator.net/))
- add `Person: ` to `PackageOriginator`
```suggestion
SPDXVersion: SPDX-2.1
DataLicense: CC0-1.0
DocumentNamespace: https://github.com/eic/epic/LICENSE.spdx-75ac9ba2-1dcd-4162-b628-e91d803eb423
PackageName: EPIC
PackageOriginator: Person: Wouter Deconinck, Sylvester Joosten
PackageHomePage: https://github.com/eic/epic
PackageLicenseDeclared: LGPL-3.0-or-later
```
This clears the errors, but there are still warnings